mock/py/mockbuild/util.py

```diff
@@ -1175,35 +1175,71 @@ def check_config(config_opts):
                                     "- option config_opts['root'] must be present in your config.")
 
 
-@traceLog()
-def include(config_file, config_opts, is_statement=False):
-    if os.path.exists(config_file):
-        if is_statement and config_file in config_opts['config_paths']:
-            getLog().warning("Multiple inclusion of %s, skipping" % config_file)
-            return
-
-        config_opts['config_paths'].append(config_file)
-
-        with open(config_file) as f:
-            content = f.read()
-            content = re.sub(r'include\((.*)\)', r'include(\g<1>, config_opts, True)', content)
-            code = compile(content, config_file, 'exec')
-        # pylint: disable=exec-used
+def _check_unrendered_config_opts(config_opts, content):
+    env = jinja2.Environment(undefined=jinja2.StrictUndefined, loader=jinja2.FileSystemLoader('/'))
+    template = env.from_string(content)
+
+    try:
+        template.render(config=config_opts)
```

Comment:
I am slightly in favour of `config_opts=config_opts`. This way the variable in a template will be the same as in config.